### PR TITLE
Add .config artifact checker

### PR DESCRIPTION
**Pull request by kees**
I wanted to learn the CI system, so I ran off and implemented a solution to the feature request I reported. ;)

### CODE REVIEW

**Review by nathanchance — APPROVED** [review]
This looks great to me! @nickdesaulniers any comments?

File: utils.py [hunk 1]
```diff
@@ -81,11 +81,14 @@ def get_cbl_name():
 
 
 def _read_builds():
+    builds = "builds.json"
+    if os.environ.get("MOCK"):
```
Comment (nickdesaulniers) [inline utils.py:85]: We should document the use of this env var in the README.

**Review by nickdesaulniers — APPROVED** [review]
great, thank you! 🛷 🌵

**Comment by roxell** [comment]
This seems like a good feature, would it be good to push this into the kernel itself?

**Pull request merged**